drill-admin/src/main/kotlin/com/epam/drill/common/AgentInfoDb.kt

```diff
@@ -10,13 +10,13 @@ import java.util.*
 
 object AgentInfos : IdTable<String>() {
     override val id: Column<EntityID<String>> = varchar("id", length = 50).primaryKey().entityId().uniqueIndex()
-    val name = varchar("name", length = 50)
+    val name = varchar("name", length = 32)
```

Comment:
No need to decrease the length of fields. Field size restrictions should be implemented through some validation functionality not related to database schema.
Moreover, there are plans to move from RDBMS to some embedded noSQL solution for config storage.